### ISSUE TITLE
Best hyper-parameters settings

### ISSUE THREAD

**Issue by lan-qing** [Justin1904/Low-rank-Multimodal-Fusion#1]
Hi Justin,

Thank you for your codes. It is helpful for me.

I want to replicate the results for research purposes. But the performance of the model can only reach about 72(F1-score on test set, MOSI dataset), while the best is 75.7 according to the paper. So could you please tell me the hyper-parameters settings in the model?

Thank you!

Best Wishes,
Lan-qing

**Comment by lan-qing** [comment]
I know some hyper-parameter settings(M = 3, d1 =32, d2 =32, d3 =64, r=4, dy =1) are shown in the paper. But there are some other hyper-parameters, including dropout-rates and  learning-rates.

I am grateful for your information.

**Comment by Justin1904** [comment]
Closed since we've discussed over emails. Will also update in the README.

**Comment by sverma88** [comment]
Hi Justin,

Can you provide me the optimal hyper-parametres as well.
I can't find the hyper-parameters in the README.

**Comment by Justin1904** [comment]
@sverma88 Sorry for the confusion. Just sent you the ones for emotion and speaker trait through email. We will need some extra effort to recover the sentiment analysis hyperparams since we lost our record on that one...
